boilerplate/app/components/switch/switch.tsx

```diff
@@ -50,73 +50,66 @@ const enhance = (style, newStyles): any => {
   return mergeAll(flatten([style, newStyles]))
```

Comment:
Someday I'd like to remove Ramda entirely. We can make a mergeAll and use `Array.prototype.flat()` to replace the few uses we're using it for.

Reply:
That sounds like its own issue - I don't think I'll do that here if that's ok.

Reply:
Agreed, just making a note of it.